# Goal         
int64 task_number
---
# Result
bool success
---
# Feedback
float64 percentage
